event-management-frontend/src/components/loginRegisterPages/Login.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
```

Comment:
I would prefer if all the logic of this component was written using Redux and Redux Saga. You have some side effects implemeed here, that would be more logically correct if they were done using saga.